_projects/zircon.md

```diff
@@ -454,19 +454,19 @@ Check the corresponding [documentation page][drawing-shapes] for more info.
 ### Fonts and tilesets
 Zircon comes with a bunch of built-in fonts tilesets. These come in 2 flavors:
 
-- CP437 tilesets *(More on using them [here][resource-handling#cp437-tilesets])*
+- CP437 tilesets *(More on using them [here]({{ site.baseurl }}{% link _zircon_docs/2018-11-22-resource-handling.md %}#cp437-tilesets))*
```

Comment:
Can you please revert these lines? `link` is broken on GitHub pages so I had to remove it.

Reply:
Done. But the original ones are not rendered either. :/